crm_stage_probability/views/crm_lead.xml

```diff
@@ -7,14 +7,14 @@
         <field name="arch" type="xml">
             <xpath
                 expr="//button[@name='action_set_automated_probability']"
-                position="before"
+                position="after"
             >
                 <field name="is_stage_probability" invisible="1" />
                 <button
                     class="ps-0 ps-md-2 btn btn-link"
```

Comment:
Could you check if this class fixes the layout.
`class="btn d-inline-block px-2 py-0 btn-link"`